README.md

```diff
@@ -1,7 +1,7 @@
 Video.js Record
 ===============
 
-Video.js plugin for recording audio/video/image files.
+Video.js plugin for recording audio/video/screen/image files.
 
```

Comment:
i guess that's true but i was hesitant to add it because 'video' can also be a virtual device streaming out a MKV movie, not just a webcam etc etc. Im +1 on leaving it as is, and grouping 'screen' into the 'video' category. There are simple too many options.

Reply:
Fixed in rebase 0200355